src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -554,17 +554,17 @@ public function nodeName()
     /**
      * Returns the node value of the first node of the list.
      *
-     * @param mixed $default When provided and the current node is empty, this value is returned and no exception is thrown
+     * @param string|null $default When not null: the value to return when the current node is empty
```

Comment:
@respinoza @amer-flix @venusdemilo could you please confirm that this covers what you were looking for with #28313?

Reply:
@nicolas-grekas works for my case! Thank you.